ui/raidboss/data/05-shb/trial/hades-ex.js

```diff
@@ -779,13 +779,13 @@ export default {
         },
       },
     },
-    { // After tanks take tower damage
```

Comment:
Could you maybe keep this comment and move it down by the delaySeconds?

Reply:
Hmmm. This seems like a bug then if the comments got dropped. Good catch!

Reply:
Did anything change to fix this happening in the future?

Reply:
No, and I'm not entirely sure on the best way to do it either; I looked at other plugins for their approach, and it seems like they generally bail if they would mess up comments.

I feel like it might be okay for the future since these should be alerting you as you're typing, not an afterthought? I guess the argument would be for people not using VSCode that get blocked by Husky on commit and accidentally remove all of their comments.

Reply:
Yeah, I think the worry is that somebody runs an `npm run lintfix` and it's destructive.  What about comments on their own line, comments on their own line after the last property, or comments on their own line before the first property?

For this case, is it possible to include the textrange between the start of node and the first property?